cob_controller_configuration_gazebo/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ set(robotlist
   cob3-9
   cob4-1
   cob4-2
+  cob4-3
+  cob4-4
```

Comment:
I changed my mind!
I think it is better to fix something whenever a bug is detected!